oslo_vmware/rw_handles.py

```diff
@@ -601,7 +621,7 @@ class VmdkReadHandle(VmdkHandle):
     """VMDK read handle based on HttpNfcLease."""
 
     def __init__(self, session, host, port, vm_ref, vmdk_path,
-                 vmdk_size, update_progress=False):
+                 vmdk_size, update_progress=True):
```

Comment:
If we set this to `True`, we call `HttpNfcLeaseProgress` on a reading connection. Is that supported?
Why do we change the default here and not in the `VmdkWriteHandle`? I'm pretty much against changing the default, because we would have to check all calling functions if they're fine with it. I think `update_progress` is supposed to be set explicitly, because the calling function might handle updates to the Nfc lease itself. E.g. Nova starts an updater thread itself (also using `update_progress()`).

Reply:
this was a debug leftover to force it on.  removing